src/recipe/parser/system_info.rs

```diff
@@ -0,0 +1,71 @@
+use serde::{Deserialize, Serialize};
+
+/// The system information for the rendered recipe.
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct SystemInfo {
+    /// `rattler-build` version.
+    rattler_build_version: String,
+    /// Git version.
+    git_version: Option<String>,
+    /// `patchelf` version (Linux).
+    patchelf_version: Option<String>,
+    /// `install_name_tool` version (macOS).
+    install_name_tool_version: Option<String>,
+}
+
+impl SystemInfo {
+    /// Collects the system information.
+    #[cfg(not(test))]
+    pub fn new() -> Self {
+        let mut system_info = SystemInfo {
+            rattler_build_version: env!("CARGO_PKG_VERSION").to_string(),
+            ..Default::default()
+        };
+        system_info.git_version = Self::run_os_command("git", "version");
+        if cfg!(target_os = "linux") {
+            system_info.patchelf_version = Self::run_os_command("patchelf", "--version");
+        } else if cfg!(target_os = "macos") {
+            system_info.install_name_tool_version =
```

Comment:
I just checked and `--version`, `-v` or `-V` flags are not available for `install_name_tool` ... will check further if there is a way

Reply:
Not even the manpage shows a version ... https://www.unix.com/man-page/osx/1/install_name_tool/

Reply:
Maybe we just have to accept this for now and store only the version. The #102 will remove the dependency anyways...

Reply:
RIP.

Reply:
We can always use [llvm-install-name-tool](https://llvm.org/docs/CommandGuide/llvm-install-name-tool.html#llvm-install-name-tool-llvm-tool-for-manipulating-install-names-and-rpaths). They implement all the cli quality of life things. And are essentially a drop-in replacement.

Reply:
and it has `--version, -V` flag!

Reply:
Good point @swarnimarun. I hope that we can complete https://github.com/prefix-dev/rattler-build/pull/442 soon and then we don't rely on `install_name_tool` at all anymore.

Reply:
Using `llvm-install-name-tool` is a good idea but I'm not sure we can *just* switch to it. It might be better to wait for #442 as Wolf said.

I removed that field in 35ad61ad687dccafa92723aa266d2890ba81e73b